diff --cc backend/src/app.module.ts
index 968c670,ea4ac75..0000000
--- a/backend/src/app.module.ts
+++ b/backend/src/app.module.ts
@@@ -4,6 -4,9 +4,12 @@@ import { join } from 'path'
  
  import { AuthModule } from './auth/auth.module';
  import { PrismaModule } from './prisma/prisma.module';
++<<<<<<< Updated upstream
++=======
+ import { ProfileModule } from './profile/profile.module';
+ import { SkillModule } from './skill/skill.module';
+ import { ProjectsModule } from './projects/projects.module';
++>>>>>>> Stashed changes
  
  @Module({
    imports: [
@@@ -13,6 -16,9 +19,12 @@@
      }),
      PrismaModule,
      AuthModule,
++<<<<<<< Updated upstream
++=======
+     ProfileModule,
+     SkillModule,
+     ProjectsModule,
++>>>>>>> Stashed changes
    ],
  })
  export class AppModule {}
* Unmerged path backend/src/profile/profile.controller.ts
* Unmerged path backend/src/profile/profile.service.ts
diff --git a/backend/prisma/schema.prisma b/backend/prisma/schema.prisma
index 6b00787..9154ab2 100644
--- a/backend/prisma/schema.prisma
+++ b/backend/prisma/schema.prisma
@@ -152,15 +152,20 @@ model ProjectMember {
 }
 
 model ProjectMedia {
-  id          Int      @id @default(autoincrement())
-  type        String   // e.g., 'IMAGE', 'VIDEO', 'LINK', 'FILE'
+  id          Int       @id @default(autoincrement())
+  type        String
   url         String
   title       String?
   description String?
-  order       Int      @default(0) // เอาไว้เรียงลำดับรูป
+  order       Int       @default(0)
 
   projectId   Int
-  project     Project  @relation(fields: [projectId], references: [id], onDelete: Cascade)
+  project     Project   @relation(fields: [projectId], references: [id], onDelete: Cascade)
+
+  deletedAt   DateTime?
+
+  @@index([projectId])
+  @@index([projectId, deletedAt])
 }
 
 // --------------------------------------
* Unmerged path backend/src/app.module.ts
* Unmerged path backend/src/profile/profile.controller.ts
* Unmerged path backend/src/profile/profile.service.ts
